crates/ty_python_semantic/src/semantic_index/use_def.rs

```diff
@@ -256,6 +256,8 @@ use crate::types::{IntersectionBuilder, Truthiness, Type, infer_narrowing_constr
 
 mod place_state;
 
+impl get_size2::GetSize for UseDefMap<'_> {}
```

Comment:
What's the reason that we use an empty implementation for `UseDefMap` given that most its fields are heap allocated?

Reply:
This was a mistake, fixed.